src/attribution/onCLS.ts

```diff
@@ -53,12 +53,7 @@ const attributeCLS = (metric: CLSMetric): CLSMetricWithAttribution => {
     }
   }
 
-  // Use Object.assign to set property to keep tsc happy.
-  const metricWithAttribution: CLSMetricWithAttribution = Object.assign(
-    metric,
-    {attribution},
-  );
-  return metricWithAttribution;
+  return {...metric, attribution};
```

Comment:
this returns a new object, though?

Reply:
Ahh, yes, thanks. I forgot I originally didn't want to do that!

Reply:
I added a comment to ensure we don't forget in the future.